code/modules/overmap/ship_weaponry/weaponry/coilgun.dm

```diff
@@ -34,7 +34,7 @@
 		var/mob/M = target
 		M.visible_message(SPAN_DANGER("<font size=5>\The [src] blows [M]'s chest apart and punches straight through!</font>"))
 	if(isturf(target) || isobj(target))
-		explosion(target, 6, 8, 10)
+		explosion(target, 6, 8, 9)
```

Comment:
Is reducing the `light_impact_range` by 1 even a meaningful change? You might want to reduce the heavy and medium too.

Reply:
> Is reducing the `light_impact_range` by 1 even a meaningful change? You might want to reduce the heavy and medium too.

Still doing some testing at the moment. It's something I've wanted to change since the last event this was featured in, though as even with the light range, it was quite wide.

Likely the other values will get lowered by 1 too after I run some testing when the map works.